### PR TITLE
adds in code-review guide from Thoughtbot

### PR DESCRIPTION
**Pull request by mejackreed**
I think this code review guide does a great job in discussing code review. Open to additional suggestions here.

### CODE REVIEW

File: code-review/README.md [hunk 1]
```diff
@@ -1 +1,78 @@
...
+* Merge once you feel confident in the code and its impact on the project.
+
+Reviewing Code
+--------------
+
+Understand why the code is necessary (bug, user experience, refactoring). Then:
+
+* Communicate which ideas you feel strongly about and those you don't.
+* Identify ways to simplify the code while still solving the problem.
+* If discussions turn too philosophical or academic, move the discussion offline
```
Comment (cbeer) [inline code-review/README.md]: I think this bullet needs editing for our current practice.

**Comment by cbeer** [comment]
:+1: Other than https://github.com/sul-dlss/DeveloperPlaybook/pull/13#discussion_r28177963, this seems pretty straightforward.

I'd like to see a section for "what do to if you're not working on a project with someone else and want to benefit from a code review", but I'll happily propose it later.

**Comment by mejackreed** [comment]
@cbeer updated with your suggestions. Thanks!

**Comment by ndushay** [comment]
:-1:  This is a older PR, pre-dating our meeting of 2015-05-21.   

While I like the doc overall, I think this is a little long based on our meeting of 2015-05-21 and I'd rather get consensus on shorter text and link out to the thoughtbot doc.   If there is consensus on this as is, I'll change my vote.  However, I would prefer something shorter with only the most salient points as our starting point.  I would also change the order of the points -- but it seems easier to vote on this as is than to do a PR to a PR and we didn't discuss a way to graciously offer alternatives.

**Comment by darrenleeweber** [comment]
:+1: although I'm on the fence about linking out or copying; there are few modifications, so just linking out and adding additional notes may suffice; on the other hand, it is easier to have it all on one page and to gain control over the content (for future modification).

**Comment by bess** [comment]
:+1:

**Comment by atz** [comment]
Now that I've read this: :+1:

**Comment by jgreben** [comment]
Since there is no cool emoji for +0 I will give it an :ok_hand:

**Comment by jmartin-sul** [comment]
:+1:

**Comment by mejackreed** [comment]
:+1: for @jgreben's :ok_hand: approach to +0

**Comment by azaroth42** [comment]
:+1: to :ok_hand: for :+0:

**Pull request merged**